package.json

```diff
@@ -172,6 +173,9 @@
         "subPermissions": [
           "settings.consortia-settings.enabled",
           "consortia.consortium.collection.get",
+          "consortia.publications.item.get",
```

Comment:
it's not right permission

it should be part of ui-consortia-settings.consortium-manager.*

Reply:
Oh right, thanks!